client/bin/pdo-invoke.psh

```diff
@@ -53,9 +54,17 @@ fi
 trap_error
 
 if -n ${params}
-   send -s _result_ -f ${pdo_file} ${method}
+   if -e ${wait} "yes"
+      send --wait -s _result_ -f ${pdo_file} ${method}
+   else
```

Comment:
Two comments on the wait flag usage:

1. wait_for_completion I believe only waits for local txn commit, is this sufficient for the multi-user tests?
2. if I look at client/controller/commands/send.py, the wait flag does not seem to be used at all. The decision to wait seems to be based on "if update_response.state_changed and commit" and not on the wait flag itself.

Reply:
Agreed. This does not fix the multi-test problem. That's going to fall into your lap (see the comments in "send"). We really need to re-think the whole CCF submitter & replication architecture.

It is still the right changes. Just need to "finish" them. Future work.